extensions/blocks/opentable/attributes.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { compact, reduce } from 'lodash';
+
+const optionValues = options => options.map( option => option.value );
+
+export const languageOptions = [
+	{ value: 'en-US', label: __( 'English-US', 'jetpack' ) },
+	{ value: 'fr-CA', label: __( 'Français-CA', 'jetpack' ) },
+	{ value: 'de-DE', label: __( 'Deutsch-DE', 'jetpack' ) },
+	{ value: 'es-MX', label: __( 'Español-MX', 'jetpack' ) },
+	{ value: 'ja-JP', label: __( '日本語-JP', 'jetpack' ) },
+	{ value: 'nl-NL', label: __( 'Nederlands-NL', 'jetpack' ) },
+	{ value: 'it-IT', label: __( 'Italiano-IT', 'jetpack' ) },
+];
+export const languageValues = optionValues( languageOptions );
+
+export const getStyleOptions = rid =>
+	compact( [
+		{ value: 'standard', label: __( 'Standard (224 x 301 pixels)', 'jetpack' ) },
```

Comment:
I wonder if the sizes in pixel here are useful or confusing.
I can see their usefulness for the widget, that is supposed to be used in narrow sidebars, but I think it's less interesting when rendered in the post content.

Reply:
I see your point. What do you think @scruffian, shall we get rid of them?

Reply:
I think we should just leave it so it aligns with their widget, and iterate later

Reply:
👍